github/github.go

```diff
@@ -222,3 +231,20 @@ func splitOwnerAndRepo(repo string) (string, string, error) {
 	}
 	return chunk[0], chunk[1], nil
 }
+
+func getEnterpriseApiUrl(baseURL string) (string, error) {
+	baseEndpoint, err := url.Parse(baseURL)
+	if err != nil {
+		return "", err
+	}
+	if !strings.HasSuffix(baseEndpoint.Path, "/") {
+		baseEndpoint.Path += "/"
```

Comment:
the end URL should be of form: https://<ENTERPRISE_URL>/api/v3

otherwise, the token endpoint is formatted to https://<ENTERPRISE_URL/api/v3//app/installations/<INSTALATION_ID>/access_tokens

recommendation is to remove the trailing / on lines 243 and on lines 246

Reply:
Interesting, as that URL formatting is directly from the client library (https://github.com/google/go-github/blob/76c3c3d7c6e78e8c91e77d2e2578c4e0a7cf96ea/github/github.go#L310-L317).

And at least it doesn't add that extra slash: https://github.com/google/go-github/blob/8e1859fece888f8f56143473c2e857a2ba84dc0c/github/apps.go#L259

Reply:
yea - I saw these as well. Let's look to see where that front loaded / comes from

Reply:
But this change (along with the slash removal) seems to fix the issue with using a github app to retrieve installation token.

Reply:
Removing last slash when returning URL from that copied function

Reply:
LGTM